### PR TITLE
Doc: new structure

### PR DESCRIPTION
**Pull request by maniyax**
Adding russian files and creating draft of new documentation structure.

Please, use new structure for documentation translate.

I will to update readme.md later, when another language of documentation will be created.

Thanks @beqabeqa473 for help with translation of single paragraphs.

### CODE REVIEW

File: doc/en/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### Speech synthesis method
+
+RHVoice uses statistical parametric synthesis. It relies on existing open source
+speech technologies (mainly [HTS](http://hts.sp.nitech.ac.jp) and related
+software). 
+
+Voices are built from recordings of natural speech. They have small footprints,
+because only statistical models are stored on users' computers. And though the
+voices lack the naturalness of the synthesizers which generate speech by
+combining segments of the recordings themselves, they are still very
```
Comment (KOLANICH) [inline doc/en/index.md:21]: May make sense to mention RNN-based synhesizers.

File: doc/ru/Privacy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+тексте: никакая часть текста или сгенерированная речь не сохраняется RHVoice.
+
+Синтез речи выполняется локально, на устройстве пользователя, без передачи
+какой-либо части текста или сгенерированной речи на удаленный сервер.
+
+Приложение RHVoice для Android использует
+[Linode Object Storage](https://www.linode.com/products/object-storage/),
+сторонний хостинг-сервис для размещения файлов языковых и голосовых данных.
+Когда вы решите установить голос, и RHVoice будет загружать необходимые файлы
+из Object Storage, Linode может собирать ваши метаданные, такие как ваш
```
Comment (KOLANICH) [inline doc/ru/Privacy.md:25]: May make sense to mention that anonimizer software may be helpful.

Comment (maniyax) [reply doc/ru/Privacy.md:25]: What? Please create your PR for changes doc/en/privacy.md.

File: doc/en/index.md (hunk 1 above)
Comment (maniyax) [reply doc/en/index.md:21]: No. It isn't readme.md. It's short information.

**Comment by maniyax** [comment]
i @alex19EP!

This issue was created two month ago...


Merge this, please.

**Review by alex19EP — CHANGES_REQUESTED** [review]
miner nit-picks. otherwise LGTM.

**Pull request merged**